mgit/index.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const meow = require( 'meow' );
+const mgit = require( './lib/index' );
+
+const cli = meow( `
+                     _ _   
```

Comment:
Haha :D Not very readable unfortunately :P Better to remove it or find a better generator (unless you designed it yourself :P).